microsoft-edge/webview2/get-started/get-started.md

```diff
@@ -6,15 +6,15 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
-ms.date: 01/24/2022
+ms.date: 02/22/2023
 ---
 # Getting Started tutorials
 
 These articles cover how to set up your development tools and create an initial WebView2 app, like a Hello World app with basic functionality.  Learn about WebView2 concepts along the way.
 
 | Article | Completed project | Description |
 |---|---|---|
-| [Get started with WebView2 in WinUI 2 (UWP) apps](winui2.md) | n/a<!--todo: upload finished project--> | Use **C# Blank App (Universal Windows)** project template and install the **Microsoft.UI.Xaml** package (WinUI 2), which installs the **Microsoft.Web.WebView2** SDK package as a dependency. |
+| [Get started with WebView2 in WinUI 2 (UWP) apps](winui2.md) | [WinUI2_GettingStarted](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/WinUI2_GettingStarted) | Use **C# Blank App (Universal Windows)** project template and install the **Microsoft.UI.Xaml** package (WinUI 2), which installs the **Microsoft.Web.WebView2** SDK package as a dependency. |
```

Comment:
This link is predicted to work after PR 172 is merged.  
PR 172 status: Approved and ready to merge.
Here is a sibling, comparable link that works now:
https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/WinUI3_GettingStarted
The link in question is:
https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/WinUI2_GettingStarted

Evidence that the link will work after 172 is merged:
Working branch for PR 172:
https://github.com/mikehoffms/WebView2Samples/tree/user/mikehoffms/getstart-winui2-initial/GettingStartedGuides/WinUI2_GettingStarted
The parent dir for that, shows 3 & 2 are parallel:
https://github.com/mikehoffms/WebView2Samples/tree/user/mikehoffms/getstart-winui2-initial/GettingStartedGuides
with 3 being:
https://github.com/mikehoffms/WebView2Samples/tree/user/mikehoffms/getstart-winui2-initial/GettingStartedGuides/WinUI3_GettingStarted
which matches the 2 link.

Reply:
https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/WinUI2_GettingStarted

That link now works, since PR 172 was merged.  @lflores-ms